docs/reporters/ConfigReporter.md

```diff
@@ -10,6 +10,7 @@ It contains:
 
 - All default MegaLinter configuration files used during the linting
 - Updated `.vscode/extensions.json` containing VsCode extensions corresponding to the linters used during MegaLinter run
+- Updated `.idea/externalDependencies.xml` containing VsCode extensions corresponding to the linters used during MegaLinter run
```

Comment:
I think it won't be VsCode extensions here

Reply:
Good catch, sorry for the typo